e2e/e2e.go

```diff
@@ -287,6 +244,23 @@ func main() { //nolint:funlen,gocyclo // Why: there are no reusable parts to ext
 	// which uses this framework -- but provisions itself.
 	if os.Getenv("SKIP_DEVENV_PROVISION") != "true" {
 		if exec.CommandContext(ctx, "devenv", "--skip-update", "status").Run() != nil {
+			var wg sync.WaitGroup
```

Comment:
Before we paralelized docker build during deployment of dependencies. Since the app and dependency deployments are merged to one devenv command, now we build docker together with provision. We might put this optimization directly to devenv.